pkg/driver/params.go

```diff
@@ -49,7 +52,11 @@ func NewVolumeParams(m map[string]string) (*VolumeParams, error) {
 	// parameter keys (not values!) are all lowercase, keys may safely be forced
 	// to the lower case.
 	m = helpers.GetCaseInsensitiveMap(&m)
-	params.VolumeGroup = m["volgroup"]
+
+	var err error
+	if params.VgRegex, err = regexp.Compile(m["volgroup"]); err != nil {
```

Comment:
shouldn't we use volgroupregex here? why we are using volgroup key here.

Reply:
With multi-pool support, I think it's not required to introduce another storage class parameter. Here we are using the same existing `volgroup` parameter, but with slight different semantics i.e now the value will be treated as regex. If the values doesn't contains any special regex characters, then existing semantics remain same (i.e exact match).